model-viewer@v3 uses `lottie-web` as a dev dependency

### Description

- I installed v3.0.0
- I use model-viewer globally (`import(/* webpackChunkName: "@google/model-viewer" */ '@google/model-viewer');`)

The error I get is:
```
Error: node_modules/@google/model-viewer/lib/features/scene-graph/api.d.ts:1:31 - error TS2307: Cannot find module 'lottie-web' or its corresponding type declarations.

1 import { AnimationItem } from 'lottie-web';
```


### Version
- model-viewer: v3.0.0


### OS
<!-- Check all that apply and please include the version tested -->
- [x] MacOS

--- a/packages/model-viewer/src/features/scene-graph/api.ts
+++ b/packages/model-viewer/src/features/scene-graph/api.ts
@@ -12,7 +12,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {AnimationItem} from 'lottie-web';
+import type {AnimationItem} from 'lottie-web';
 
 import {AlphaMode, MagFilter, MinFilter, WrapMode} from '../../three-components/gltf-instance/gltf-2.0.js';
 

--- a/packages/model-viewer/src/features/scene-graph/image.ts
+++ b/packages/model-viewer/src/features/scene-graph/image.ts
@@ -13,7 +13,7 @@
  * limitations under the License.
  */
 
-import {AnimationItem} from 'lottie-web';
+import type {AnimationItem} from 'lottie-web';
 import {Mesh, MeshBasicMaterial, OrthographicCamera, PlaneGeometry, Scene, Texture as ThreeTexture, WebGLRenderTarget} from 'three';
 
 import {blobCanvas} from '../../model-viewer-base.js';

